### PR TITLE
Add link to examples in tutorial

### PR DESCRIPTION
**Pull request by crcrpar**
## Motivation
Add link to examples in tutorial as other resource.

### CODE REVIEW

File: tutorial/README.rst [hunk 1]
```diff
@@ -13,5 +13,13 @@ The order we assume is as follows:
 * :ref:`pruning`
 * :ref:`sampler`
 
+Other Resources
```
Comment (hvy) [inline tutorial/README.rst:16]: Just curious but are you expecting other items besides "examples" under this header?

File: tutorial/README.rst [hunk 2]
```diff
@@ -13,5 +13,13 @@ The order we assume is as follows:
...
+===============
+
+* `examples`_
```
Comment (hvy) [inline tutorial/README.rst:19]: How about capitalizing the "e"?

**Comment by HideakiImamura** [comment]
Thank you for the PR. I left a minor note: this PR seems to be relevant to Issue #1617.

File: tutorial/README.rst (hunk 1 above)
Comment (crcrpar) [reply tutorial/README.rst]: imho, the conference talk videos are candidates.

File: tutorial/README.rst [hunk 3]
```diff
@@ -13,5 +13,11 @@ The order we assume is as follows:
 * :ref:`pruning`
 * :ref:`sampler`
 
+**Other Resources**:
+
+* `Examples`_: Demonstarate how to use Optuna with popular libraries for machine learning and deep learning.
```
Comment (hvy) [inline tutorial/README.rst:18]: A details, but those examples are not limited to integrations (e.g. the quadratic function example). How about something along "For more examples including how to use Optuna with... "?

**Pull request merged**